MobileCenterDistribute/MobileCenterDistributeTests/MSDistributeTests.m

```diff
@@ -783,21 +796,22 @@ - (void)testDontPersistLastestReleaseIfNotMandatory {
 - (void)testOpenUrl {
 
   // If
+  NSString *requestId = @"FIRST-REQUEST";
+  NSString *token = @"TOKEN";
   NSString *scheme = [NSString stringWithFormat:kMSDefaultCustomSchemeFormat, kMSTestAppSecret];
   id distributeMock = OCMPartialMock(self.sut);
+  OCMReject([distributeMock checkLatestRelease:OCMOCK_ANY releaseHash:OCMOCK_ANY]);
   OCMStub([distributeMock sharedInstance]).andReturn(distributeMock);
-  OCMStub([distributeMock checkLatestRelease:[OCMArg any] releaseHash:kMSTestReleaseHash]).andDo(nil);
   id mobileCeneterMock = OCMClassMock([MSMobileCenter class]);
   OCMStub([mobileCeneterMock isConfigured]).andReturn(YES);
-  [self mockMSPackageHash];
+  id utilityMock = [self mockMSPackageHash];
   
   // When
   NSURL *url = [NSURL URLWithString:[NSString stringWithFormat:@"%@://?", scheme]];
   BOOL result = [MSDistribute openURL:url];
   
   // Then
   assertThatBool(result, isFalse());
-  OCMReject([distributeMock checkLatestRelease:[OCMArg any] releaseHash:[OCMArg any]]);
```

Comment:
There are 6 conditions that `checkLatestRelease:releaseHash:` is not being called and this test is verifying that. Can you tell me why you remove all OCMReject here?